tests/integration/multicluster/cluster_local.go

```diff
@@ -1,4 +1,4 @@
-// Copyright Istio Authors
+// Copyright 2019 Istio Authors
```

Comment:
nit: Remove dates, new files shouldn't have them. We haven't done a good job at scrubbing old files.